code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -853,6 +902,7 @@
 /datum/tarot/reversed/the_stars
 	name = "XVII - The Stars?"
 	desc = "May your loss bring fortune."
+	extended_desc = "will deal heavy genetic damage to the user, as well as giving a random limb either a fracture, IB, or a burn wound. However the user will also draw two additional cards."
```

Comment:
```suggestion
	extended_desc = "will deal heavy genetic damage to the user as well as giving a random limb either a fracture, internal bleeding, or a burn wound. However, the user will also have to draw two additional cards."
```

A comma before "as well as" should be used if the second half of the sentence is less important than the first one